disposebag/src/main/kotlin/com/kizitonwose/android/disposebag/DisposableExtensions.kt

```diff
@@ -5,12 +5,10 @@ import android.arch.lifecycle.Lifecycle
 import android.arch.lifecycle.LifecycleOwner
 import io.reactivex.disposables.Disposable
 
-/**
- * Created by Kizito Nwose
- */
+fun Disposable.disposedBy(bag: DisposeBag) = bag.add(this)
```

Comment:
I don't think this extension belongs in the same scope as the Lifecycle owner extension. They're all Disposable extensions but keeping it in the same file with the DisposeBag class is better, for visibility.